compose-cryostat.yaml

```diff
@@ -0,0 +1,203 @@
+      - ${PWD}/conf:/opt/cryostat.d/conf.d
+      - ${PWD}/templates:/opt/cryostat.d/templates.d
+      - ${PWD}/truststore:/truststore
+      - ${PWD}/probes:/opt/cryostat.d/conf.d/probes.d
+      - ${XDG_RUNTIME_DIR}/podman/podman.sock:/run/user/0/podman/podman.sock:Z
+    security_opt:
+      - label=disable
+
+    environment:
+      - CRYOSTAT_ENABLE_JDP_BROADCAST=true
+      - CRYOSTAT_REPORT_GENERATOR=http://cryostat-reports:10001
+      - CRYOSTAT_PLATFORM=io.cryostat.platform.internal.PodmanPlatformStrategy,io.cryostat.platform.internal.DefaultPlatformStrategy
+      - CRYOSTAT_DISABLE_SSL=${CRYOSTAT_DISABLE_SSL}
+      - CRYOSTAT_DISABLE_JMX_AUTH=${CRYOSTAT_DISABLE_JMX_AUTH:-false}
+      - CRYOSTAT_ALLOW_UNTRUSTED_SSL=true
+      - CRYOSTAT_RJMX_USER=smoketest
+      - CRYOSTAT_RJMX_PASS=smoketest
+      - CRYOSTAT_RJMX_PORT=9091
+      - CRYOSTAT_RMI_PORT=9091
+      - CRYOSTAT_CORS_ORIGIN=${CRYOSTAT_CORS_ORIGIN}
+      - CRYOSTAT_WEB_HOST=localhost
+      - CRYOSTAT_WEB_PORT=8282
+      - CRYOSTAT_EXT_WEB_PORT=8282
+      - CRYOSTAT_MAX_WS_CONNECTIONS=${CRYOSTAT_MAX_WS_CONNECTIONS}
+      - CRYOSTAT_AUTH_MANAGER=io.cryostat.net.BasicAuthManager
+      - CRYOSTAT_TARGET_MAX_CONCURRENT_CONNECTIONS=${CRYOSTAT_TARGET_MAX_CONCURRENT_CONNECTIONS}
+      - CRYOSTAT_TARGET_CACHE_TTL=${CRYOSTAT_TARGET_CACHE_TTL}
+      - CRYOSTAT_CONFIG_PATH=/opt/cryostat.d/conf.d
+      - CRYOSTAT_ARCHIVE_PATH=/opt/cryostat.d/recordings.d
+      - CRYOSTAT_TEMPLATE_PATH=/opt/cryostat.d/templates.d
+      - CRYOSTAT_PROBE_TEMPLATE_PATH=/opt/cryostat.d/conf.d/probes.d
+      - CRYOSTAT_CLIENTLIB_PATH=/clientlib
+      - CRYOSTAT_REPORT_GENERATION_MAX_HEAP=200
+      - CRYOSTAT_DISCOVERY_PING_PERIOD=${CRYOSTAT_DISCOVERY_PING_PERIOD}
+      - CRYOSTAT_ACTIVE_REPORTS_CACHE_EXPIRY_SECONDS=${CRYOSTAT_ACTIVE_REPORTS_CACHE_EXPIRY_SECONDS}
+      - CRYOSTAT_ACTIVE_REPORTS_CACHE_REFRESH_SECONDS=${CRYOSTAT_ACTIVE_REPORTS_CACHE_REFRESH_SECONDS}
+      - CRYOSTAT_PUSH_MAX_FILES=${CRYOSTAT_PUSH_MAX_FILES}
+      - CRYOSTAT_VERTX_POOL_SIZE=${CRYOSTAT_VERTX_POOL_SIZE}
+      - GRAFANA_DATASOURCE_URL=http://jfr-datasource:8080
+      - GRAFANA_DASHBOARD_EXT_URL=http://localhost:3000
+      - GRAFANA_DASHBOARD_URL=http://grafana:3000
+      - KEYSTORE_PATH=${KEYSTORE_PATH}
+      - KEYSTORE_PASS=${KEYSTORE_PASS}
+      - KEY_PATH=${KEY_PATH}
+      - CERT_PATH=${CERT_PATH}
+      - CRYOSTAT_JUL_CONFIG=${CRYOSTAT_JUL_CONFIG}
+      - CRYOSTAT_JDBC_DRIVER=org.h2.Driver
+      - CRYOSTAT_JDBC_URL=jdbc:h2:mem:cryostat;DB_CLOSE_DELAY=-1;INIT=create domain if not exists jsonb as varchar
+      - CRYOSTAT_JDBC_USERNAME=cryostat
+      - CRYOSTAT_HIBERNATE_DIALECT=org.hibernate.dialect.H2Dialect
+      - CRYOSTAT_HBM2DDL=create
+      - CRYOSTAT_JMX_CREDENTIALS_DB_PASSWORD=smoketest
+      - CRYOSTAT_LOG_DB_QUERIES=true
+      - CRYOSTAT_DEV_MODE=true
+    expose:
+      - "9091"
+    ports:
+      - "8282:8282"
+    profiles: ["duplicate"]
+
+  # Testing apps
+  scaled-app:
```

Comment:
I'm not sure if I am just doing it wrong, but it seems like these always get included in the deployment for me, even without `-i`/`CT_EN_INVALID`. I am using podman and podman-compose, and just trying ex. `bash compose-test.bash` or `bash compose-test.bash -a 5 -j 5`

Reply:
Oh, I haven't added those to be able to be configured off yet, thanks for noticing.